From eaa8b48e3d84d1ac7bdb7764024172844ab20a98 Mon Sep 17 00:00:00 2001
Message-ID: <eaa8b48e3d84d1ac7bdb7764024172844ab20a98.1724198118.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1724198118.git.sharlatanus@gmail.com>
References: <cover.1724198118.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 30 Jul 2024 22:21:15 +0100
Subject: [PATCH 07/32] gnu: python-czml3: Update to 1.0.2.

* gnu/packages/astronomy.scm (python-czml3): Update to 1.0.2.
[native-inputs]: Remove python-pytest-cov.

Change-Id: I580ccbe9f87e6ba2d01e4181d4e491640c6dcf09
---
 gnu/packages/astronomy.scm | 13 ++++---------
 1 file changed, 4 insertions(+), 9 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 5ff5e02811..91271e665f 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -2396,7 +2396,7 @@ (define-public python-crds
 (define-public python-czml3
   (package
     (name "python-czml3")
-    (version "1.0.1")
+    (version "1.0.2")
     (source
      (origin
        (method git-fetch)
@@ -2405,17 +2405,12 @@ (define-public python-czml3
              (commit (string-append "v" version))))
        (file-name (git-file-name name version))
        (sha256
-        (base32 "1vm9ajpnwxncvsl1ix3jarqyi14b0703g12wsr8b0agkrjlpmvmx"))))
+        (base32 "0gsdffn5w7j1aridvmx4xh6axvmbmqjwawacc8fnshpcyh4md5pn"))))
     (build-system pyproject-build-system)
     (propagated-inputs
-     (list python-attrs
-           python-dateutil
-           python-w3lib))
+     (list python-attrs python-dateutil python-w3lib))
     (native-inputs
-     (list python-astropy
-           python-pytest
-           python-pytest-cov
-           python-pytest-mypy))
+     (list python-astropy python-pytest python-pytest-mypy))
     (home-page "https://github.com/poliastro/czml3")
     (synopsis "Python library to write CZML")
     (description
-- 
2.41.0

